cypress/automated-tests/youth-pass/parental-consent.cy.js

```diff
@@ -0,0 +1,42 @@
+import { faker } from '@faker-js/faker';
+import { getRandomApplicantAge } from '../../common/birthdate-constants';
+
+describe('youth pass parental consent flow', () => {
+  it('proceeds through an application', () => {
+    const youthPassUrl = Cypress.env('youth_pass_url');
+    const applicantZipCode = '02114';
+    
+    cy.visit(youthPassUrl);
+    cy.get('#form-section-0 > .form-section-buttons > .form-section-next').click();
+    cy.get('#form-section-1 > .form-section-buttons > .form-section-next').click();
+    cy.get('#element105').type(applicantZipCode).blur();
+    
+  });
+
+  it("shows the parental consent fields only when under 13", function() {
+    const applicantBirthdate13 = getRandomApplicantAge().        applicantBirthday13;
+    const applicantBirthdate12 = getRandomApplicantAge().        applicantBirthday12;
```

Comment:
Question: Is the spacing after the function call on lines 17-18 intentional?

Reply:
Nope, that's weird. Thanks.